api/events/msam-events.json

```diff
@@ -17,7 +17,8 @@
                     "Statement": [{
                         "Effect": "Allow",
                         "Action": [
-                            "dynamodb:PutItem"
+                            "dynamodb:PutItem",
+                            "medialive:Describe*"
```

Comment:
@morjoan I recall you mentioning that this file is no longer being used? 
For my own sanity, I still added this to the file just in case.

Reply:
That's correct. If you look at the build.sh script, we go from the yml file to the release version of this file. No need to modify this either.

Reply:
Correct. If you look at the build.sh script, you'll see that we write straight to the msam-events-release.json.

Reply:
ok. this has been removed as well